src/lib/crypto/kimchi_backend/gadgets/range_check.ml

```diff
@@ -2,16 +2,23 @@ open Core_kernel
 
 open Kimchi_backend_common.Plonk_constraint_system.Plonk_constraint
 
-(* 64-bit range-check gadget *)
-let range_check64 (type f)
+(* Helper to create RangeCheck0 gate, configured in various ways
+ *     - is_64bit   : create 64-bit range check
+ *     - is_compact : compact limbs mode (only used by compact multi-range-check)
```

Comment:
somewhat unrelated, but because I've seen this a lot on the Rust side and I think it was breaking the spec, so pointing it out here, indented lists are not valid markdown (and there should be a line break before the list)

Reply:
I was not intending this to be markdown.  Is something translating it to MD?

Reply:
This should preferably be [ocamldoc](https://v2.ocaml.org/manual/ocamldoc.html#) syntax, not markdown. As such, it looks okay.